client/layout/masterbar/oauth-client.scss

```diff
@@ -26,15 +25,19 @@
 	}
 }
 
+.masterbar__oauth-client-logo img {
+	max-height: 46px;
```

Comment:
As is, max-height of 46px will put a blue border underneath the logo (due to `.masterbar`'s `height: 46px; border-bottom: 1px solid #0079aa;`). 

Should we use 47px instead to cover the blue bottom border?

(This is especially obvious with white background logos, like IntenseDebate.)

Reply:
That's a good point indeed. I saw that problem during implementation and tried to set a `box-shadow` of `1px` at the bottom of the logo to be consistent with this blue border. Unfortunately, I couldn't find a way to make it work. I've just tried again to no avail, so I'll follow your suggestion for the time being. Maybe @danhauk can circle back on this when he has a few.